app/src/features/mocks/screens/mocksList/components/MocksList/MocksList.tsx

```diff
@@ -8,11 +8,24 @@ import {
   redirectToMockEditorEditMock,
 } from "utils/RedirectionUtils";
 import { trackMockUploadWorkflowStarted, trackNewMockButtonClicked } from "modules/analytics/events/features/mocksV2";
-import { MockListSource, MockType, RQMockMetadataSchema } from "components/features/mocksV2/types";
+import {
```

Comment:
![CleanShot 2024-04-30 at 15 20 12](https://github.com/requestly/requestly/assets/16779465/c8a9fa5b-66ca-4151-91ef-b4439c9576ab)

- Select shown on mock collection too in picker modal
- Styling breaking